### DISCUSSION TITLE
zlibrary上似有人发布了中国当代政治运动史数据库（宋永毅）的最新版

### DISCUSSION THREAD

**Discussion by TonokoTakatsuki** [banned-historical-archives/banned-historical-archives.github.io#2940] [General]
对照了官网的更新列表看了下内容，无论是文革数据库还是大跃进数据库应都为2023年2月后的最新版本。

**Comment by banned-historical-archives** [comment]
@TonokoTakatsuki 我们已经录入了中国当代政治运动史数据库 https://github.com/banned-historical-archives/CCRD

**Reply by banned-historical-archives** [reply]
https://banned-historical-archives.github.io/articles/  你可以在来源过滤器中找到这四个子数据库

**Reply by TonokoTakatsuki** [reply]
好的了解，辛苦你们了
